aiohttp/web_response.py

```diff
@@ -408,6 +408,9 @@ def write(self, data):
         assert not self._eof_sent, "EOF has already been sent"
         assert self._payload_writer is not None, \
             "Response has not been started"
+        warnings.warn("drain method is deprecated, use await resp.write()",
```

Comment:
`write` method should be changed to async. Otherwise this suggestion will lead to errors.

Reply:
Please elaborate

Reply:
`StreamResponse.write` is not `async` now. What it returns is a mystery. According the interface `StreamResponse.write` breaks it. So it would be fine to expect strange things including TypeErrors on trying to await non-coroutine value.

Reply:
`write` is async now.